### PR TITLE
Carousel css tweaks

### PR DESCRIPTION
**Pull request by axdanbol**
Description of changes - 

@mentions of the person or team responsible for reviewing proposed changes.

TO-DO checklist before submitting your pull-request - 

- [x] Are all acceptance criteria met ? 
- [x] Has the code been documented properly ?
- [x] Does component css-class structure conform to the Blue Dino style template i.e. top-level element .wrapper, .container and other inside css classes ?
- [x] Have all checks passed ?
- [x] Have you tested inter-browser functionality (should work on the latest Chrome and Firefox browsers) ?

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -7,6 +7,10 @@ body {
 
 * {
```
Comment (GagsdName) [inline src/styles.scss]: formatting is off in this file. Please review.

File: src/app/components/carousel/carousel.component.scss [hunk 2]
```diff
@@ -26,21 +26,21 @@
       }
     }
 
-    &:hover svg {
-      filter: drop-shadow(0px 4px 8px #000000);
-      opacity: 0.6;
-      transition: opacity 0.5s;
+    // &:hover svg {
```
Comment (GagsdName) [inline src/app/components/carousel/carousel.component.scss]: remove commented out code ?

Comment (axdanbol) [reply src/app/components/carousel/carousel.component.scss]: I don't know if we might want to re-enable this in the future?

File: src/styles.scss (hunk 1 above)
Comment (axdanbol) [reply src/styles.scss]: Fixed the styling. FYI I wasn't the one that messed it up.

Comment (GagsdName) [reply src/styles.scss]: I understand, it was very noticeable so pointed it out.
Please insert a newline after the imports too.

File: src/app/components/carousel/carousel.component.scss (hunk 2 above)
Comment (GagsdName) [reply src/app/components/carousel/carousel.component.scss]: We can add it then, I think.

**Pull request merged**